.github/workflows/canary-ubuntu.yml

```diff
@@ -79,9 +72,39 @@ jobs:
           DJL_ENGINE=tflite-native-auto ./gradlew clean run
           rm -rf ~/.djl.ai
           DJL_ENGINE=tflite-native-cpu ./gradlew clean run
+          rm -rf ~/.djl.ai
       - name: Test PaddlePaddle
         working-directory: canary
         run: |
           DJL_ENGINE=paddlepaddle-native-auto ./gradlew clean run
           rm -rf ~/.djl.ai
           DJL_ENGINE=paddlepaddle-native-cpu ./gradlew clean run
+          rm -rf ~/.djl.ai
+      - name: Test Xgboost
+        working-directory: canary
+        run: |
+          DJL_ENGINE=xgboost ./gradlew clean run
+
+  canary-mxnet-ubuntu18_04:
```

Comment:
Why making MXNet as a single job?

Reply:
mxnet-native-mkl and cu102, cu110 shared library copied from pip package, they only support 18.04,
I rebulit mxnet-native-auto, only auto support ubuntu 16.04